src/ucs/arch/x86_64/cpu.c

```diff
@@ -485,6 +485,16 @@ ucs_cpu_model_t ucs_arch_get_cpu_model()
                 break;
             }
             break;
+        /* AMD Zen5 */
+        case 0x1a:
+            switch (model) {
+            case 0x00 ... 0x2f:
+            case 0x40 ... 0x4f:
+            case 0x60 ... 0x7f:
```

Comment:
seems it's a non-standard extension? maybe use if condition?

Reply:
If I use the 'if condition' for 'model', it would make the code less structured. The 'case ranges' are a well-known extension supported by both GCC and Clang. Can we stick with it?

Reply:
i'd prefer if/else

Reply:
I will change the style then.